mesonbuild/dependencies/cmake.py

```diff
@@ -637,7 +637,7 @@ def get_variable(self, *, cmake: T.Optional[str] = None, pkgconfig: T.Optional[s
                 return ';'.join(v)
         if default_value is not None:
             return default_value
-        raise DependencyException(f'Could not get cmake variable and no default provided for {self!r}')
+        raise DependencyException(f'Could not get {self.name} cmake variable {cmake} and no default provided')
```

Comment:
Should we use e.g. `!r` for the variable name (not necessarily self.name) in order to distinguish the variable name via quoting?